.env.DEV

```diff
@@ -9,3 +9,4 @@ USE_CDN=
 API_KEY='example-application-id'
 API_SECRET='example-secret'
 COOKIE_SECRET='b8f6959746b617d9c9e90ce4a4b6e0'
+SCRIPT_BASE=scripts
```

Comment:
I'm confused by this, why is it searching under '/'? Doesn't listFiles() early-return when this variable isn't set?

Reply:
I think I got confused, initially when I didn't set `SCRIPT_BASE` and got no detected scripts I read too quickly the code and thought that it was normal because it was looking at `/.`.

I've removed the commit; there is another discussion over at #52 where it might be needed but it depends on how we want to manage WCIF data so I'll include it there if we need it in the end.